### ISSUE TITLE
How to remove "Return to Pantheon" customization on WordPress login form

### ISSUE THREAD

**Issue by ari-gold** [pantheon-systems/documentation#3538]
Re: <Doc title and/or link>
## Issue Description

We added a customized button to help with the first time user experience. Some customers would prefer to not have this shown. Let's document how to remove it.

ref. https://github.com/pantheon-systems/WordPress/pull/137

## Suggested Resolution

Put this code somewhere?
```
/**
 * Remove reutrn to Pantheon button
 */
function remove_return_to_Pantheon_button_scripts_and_styles() 
{

    wp_dequeue_style('pantheon-login-mods');

    wp_dequeue_script('pantheon-login-mods');

}

add_action( 'login_enqueue_scripts', 'remove_return_to_Pantheon_button_scripts_and_styles', 25 );

function disable_return_to_Pantheon_button()
{

    remove_action('login_header', 'Return_To_Pantheon_Button_HTML');

    remove_action('login_enqueue_scripts', 'Pantheon_Enqueue_Login_script');

    remove_action('login_enqueue_scripts', 'Pantheon_Enqueue_Login_style');
}

add_action( 'after_setup_theme', 'disable_return_to_Pantheon_button', 10 );
```

**Comment by jamesstcrice** [comment]
Thanks for this documentation, Ari.

I'd vote that this feature (the button) is not necessary, given that Pantheon is generally geared toward more technical users. It essentially amounts to advertising in my opinion. In my case, I see it potentially confusing client users who have nothing to do with Pantheon administration.

Thanks

**Comment by ari-gold** [comment]
Hi @jamesstcrice  -- thanks for chiming in. Please also see more discussion on https://github.com/pantheon-systems/WordPress/issues/153

**Comment by jamesstcrice** [comment]
Thanks, Ari. I only added it to one site so far, which "belongs" to a particularly basis client, but I created a plugin that disables the button using your lines of code.

**Comment by ataylorme** [comment]
@jamesstcrice @ari-gold please use this code:

```
/**
 * Remove reutrn to Pantheon button
 */
function remove_return_to_Pantheon_button_scripts_and_styles() 
{

    wp_dequeue_style('pantheon-login-mods');

    wp_dequeue_script('pantheon-login-mods');

}

add_action( 'login_enqueue_scripts', 'remove_return_to_Pantheon_button_scripts_and_styles', 25 );

function disable_return_to_Pantheon_button()
{

    remove_action('login_header', 'Return_To_Pantheon_Button_HTML');

    remove_action('login_enqueue_scripts', 'Pantheon_Enqueue_Login_script');

    remove_action('login_enqueue_scripts', 'Pantheon_Enqueue_Login_style');
}

add_action( 'after_setup_theme', 'disable_return_to_Pantheon_button', 10 );
```

Tested [here](http://no-button-return-to-pantheon-test.pantheonsite.io/wp-login.php).

**Comment by jamesstcrice** [comment]
@ataylorme Thanks, will do

**Comment by rachelwhitton** [comment]
Closed by pantheon-systems/WordPress#155
